airflow/utils/context.pyi

```diff
@@ -102,24 +102,19 @@ class Context(TypedDict, total=False):
     ds: str
     ds_nodash: str
     exception: BaseException | str | None
-    execution_date: DateTime
     expanded_ti_count: int | None
     inlets: list
     inlet_events: InletEventsAccessors
     logical_date: DateTime
     macros: Any
     map_index_template: str
-    next_ds: str | None
-    next_ds_nodash: str | None
-    next_execution_date: DateTime | None
+    next_logical_date: DateTime | None
```

Comment:
Same

Reply:
I have removed: prev_logical_date_success, prev_logical_date and next_logical_date.